zbuf/merger.go

```diff
@@ -17,6 +17,17 @@ func NewComparator(zctx *zed.Context, layout order.Layout) *expr.Comparator {
 	return expr.NewComparator(nullsMax, !nullsMax, exprs...).WithMissingAsNull()
 }
 
+func NewComparatorNullsMax(zctx *zed.Context, layout order.Layout) *expr.Comparator {
```

Comment:
Am I missing something or is the behavior of this identical to that of NewComparator?

Reply:
Oh yeah meant to fix this

Reply:
Actually, I am futzing with this.  The merge op unit test doesn't seem to work with the NullsMax variation and there's no way to use NewComparator in a way where nullsmax is always true and reverse is true only for order.Desc.  I propose we merge this is and clean it up later.

Reply:
Ah, I totally missed that the first argument to expr.NewComparator here is always true.